lib/git_reflow/sandbox.rb

```diff
@@ -14,10 +14,12 @@ def run(command, options = {})
 
       if options[:with_system] == true
         system(command)
-      elsif options[:loud] == true
-        puts `#{command}`
       else
-        `#{command}`
+        output = `#{command}`
+        if options[:loud] == true
```

Comment:
I'm not sure I like the nested `if` in here.  is there a reason you added this over the `elsif` above?